packages/components/src/keyboard-shortcuts/test/index.js

```diff
@@ -1,32 +1,27 @@
 /**
  * External dependencies
  */
-import { mount } from 'enzyme';
+import { fireEvent, render, screen } from '@testing-library/react';
 
 /**
  * Internal dependencies
  */
 import KeyboardShortcuts from '../';
 
 describe( 'KeyboardShortcuts', () => {
-	afterEach( () => {
-		while ( document.body.firstChild ) {
-			document.body.removeChild( document.body.firstChild );
-		}
-	} );
-
 	function keyPress( which, target ) {
 		[ 'keydown', 'keypress', 'keyup' ].forEach( ( eventName ) => {
 			const event = new window.Event( eventName, { bubbles: true } );
 			event.keyCode = which;
 			event.which = which;
-			target.dispatchEvent( event );
+			fireEvent( target, event );
```

Comment:
It would be great if we could follow-up with a refactor from `fireEvent` to `user-event` (and from `keyCode` to `code` for keyboard events)

Reply:
Good call. While I'm not prioritizing that refactor at this time, I'm going to add it to my list so it doesn't fall through the cracks.